index.bs

```diff
@@ -279,5 +512,149 @@ Threat model {#security-threat}
 Mitigations and defenses {#security-defenses}
 ---------------------------------------------
 
+Appending A: Messages
+=====================
+
+Lower type keys are used for messages we expect to be more frequent
+and higher type keys are used for less frequently used messages.  This
+is because on the wire lower values encode smaller.
+
+<pre>
+; type key 14
+presentation-url-availability-request = {
+  request
+  1: [* text] ; urls
```

Comment:
Nit: is an empty list valid here?  Similar question for presentation-url-availability-event

Reply:
Sure, but the response would just be an empty list.

For the event, it would tell you nothing is available any more.

Reply:
Why would a controller ever want to send an empty list?

Is the empty list behavior for the event documented?

Reply:
It wouldn't want to send an empty list.  There's no reason.  The behavior would have to be to return a response with an empty list.  

I just added a "must not be empty" to make it clear this must never happen.